public/css/gogs.css

```diff
@@ -1311,6 +1311,8 @@ footer .ui.language .menu {
 .repository.file.list #file-content .code-view .lines-num .hljs li,
 .repository.file.list #file-content .code-view .lines-code .hljs li {
   padding-left: 5px;
+  display: inline-block;
```

Comment:
It appears that the whole file is two space indented, rather than tabs. See [here](https://raw.githubusercontent.com/gogits/gogs/master/public/css/gogs.css). I don't mind either way but currently it seems to match the rest of the file.